#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_more_warnings.dpatch by Petter Reinholdtsen

Make more issues visible by using -W in addition to -Wall.

@DPATCH@
--- insserv-1.11.0.orig/Makefile
+++ insserv-1.11.0/Makefile
@@ -28,7 +28,7 @@
 	  COPTS = -O2
 endif
 endif
-	 CFLAGS = -Wall $(COPTS) $(DEBUG) $(LOOPS) -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
+	 CFLAGS = -W -Wall $(COPTS) $(DEBUG) $(LOOPS) -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 		  $(ISSUSE) -DINITDIR=\"$(INITDIR)\" -DINSCONF=\"$(INSCONF)\" -pipe
 	  CLOOP = -falign-loops=0
 	     CC = gcc
